pkg/login/social/util.go

```diff
@@ -0,0 +1,57 @@
+package social
```

Comment:
NIT: I don't think we need both a util and common file. Maybe put this is common.go?

Reply:
Thanks, done!